test/functionalTest/cases/2763_postUpdateContext_error_return/invalid_service_path_gives_400_bad_request.test

```diff
@@ -0,0 +1,56 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
There wasn't any, and the problem wasn't even fixed.
I just assumed (and forgot about the functest), that the low level parse functions would take care of this (as they should). However subscriptions use their own implementation of entity parsing ...

Fixed in XXX